src/screens/help.ts

```diff
@@ -178,7 +181,9 @@ export default async function helpScreen(): Promise<string> {
 		}
 
 		if (mergedSpec.data.accepts) {
-			fullDescription += chalk.gray.italic(` (accepts: ${mergedSpec.data.accepts.join(', ')})`);
+			if (typeof mergedSpec.data.accepts !== 'function') {
+				fullDescription += chalk.gray.italic(` (accepts: ${mergedSpec.data.accepts.join(', ')})`);
+			}
```

Comment:
Why is this wrapped in a `(typeof mergedSpec.data.accepts !== 'function')` conditional?

Reply:
By the time this code executes, mergedSpec.data.accepts will have already been resolved to an array, but, TypeScript doesn't know that, so, this is a type guard effectively, just to make TypeScript happy.

Reply:
I suppose, it could all be within the single outer `if`

Reply:
Does `mergedSpec` have an inaccurate type, then? If `mergedSpec` always has an array and the function has always been called, then it sounds like it has an outdated/inaccurate type.

Reply:
`mergedSpec` sort of has an issue in that until the values are resolved by potentially having to call a function, there is some uncertainty as to what the `accepts` value really is.  Currently, the `data.accepts` is being resolved fairly early, and is being done in one place.  Thereafter, it will always have been resolved, and perhaps having a variant of the type that reflects this would make things simpler. But, that's not the case for the `options[key].accepts`, which is resolved only when needed.  Unless that's changed, this needs to stay as it is.

Reply:
Okay, thanks for explaining!

Reply:
Would you be interested in coercing the returned type?

The 'upside' is that it will remove confusion from outside of `get-command-spec.ts` regarding whether the `data?.accepts?` could be a function call or not (currently, this *never* happens).

We can have a definition that's exposed that's derived.  What it needs is for `get-command-spec.ts` to have the following:

```ts
type ResolvedData = Omit<Pick<Required<UnresolvedGenericCommandSpec>, 'data'>['data'], 'accepts'> & {
	accepts?: string[] | number[];
};
```
to redefine the `data?.accepts?` and add back `data?.accepts?` and expose it with:
```ts
export type GenericCommandSpec = Omit<UnresolvedGenericCommandSpec, 'data'> & { data?: ResolvedData };
```

and of course, the coercion:
```ts
return spec as GenericCommandSpec;
```

I suppose, the inverse would be possible too: ie: define `GenericCommandSpec` without the function calls, and expose that, but, use the modified version internally, that adds the callback function possibilities.

Either way, it would clarify that outside of `get-command-spec.ts`, that this will only have arrays, and never a callback function.

Reply:
Seems that the inverse is way simpler.

Reply:
I think it's okay how it is, but maybe we can consider that in a future PR.